tests/p_make/10-test_make.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Author: Lz <Lz843723683@163.com>
+
+if [ "$centos_ver" -ge "8" ];then
```

Comment:
as gcc and make are available, can you (if you still want to) rebase this PR to also cover 8 ?